README.md

```diff
@@ -1 +1,188 @@
-# awesome-book
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Awesome book] <a name="about-project"></a>
+
+> In this project we created and access properties and methods of JavaScript objects.
+
+**[Wesome books]**
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> plain javascript
+
+<details>
+  <summary>Client</summary>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Add_book]**
+- **[Display_book]**
+- **[Remove_book]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> clone the repo
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+
+```sh
+ git clone (repo name) 
+```
+ 
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+ git clone (repo name) 
+```
+ 
+
+### Install
+
+Install this project with:
+
+```sh
+ git clone (repo name) 
+```
+ 
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+ git clone (repo name) 
+```
+ 
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+ open in browser 
+```
+ 
+
+### Deployment
+
+You can deploy this project using:
+
+```sh
+ open in browser 
+```
```

Comment:
- [ ] Please update these sections accordingly how to install clone and run your project